src/app/component-library/catalog/asset-detail-dialog/asset-property-grid-group-builder.ts

```diff
@@ -123,60 +112,41 @@ export class AssetPropertyGridGroupBuilder {
       fields.push({
         icon: 'commute',
         label: 'Transport Mode',
-        labelTitle: AssetProperties.transportMode,
         ...this.propertyGridUtils.guessValue(asset.transportMode?.label),
       });
     }
     if (includeEmpty || asset.dataCategory) {
       fields.push({
         icon: 'commute',
         label: 'Data Category',
-        labelTitle: AssetProperties.dataCategory,
         ...this.propertyGridUtils.guessValue(asset.dataCategory?.label),
       });
     }
     if (includeEmpty || asset.dataSubcategory) {
       fields.push({
         icon: 'commute',
         label: 'Data Subcategory',
-        labelTitle: AssetProperties.dataSubcategory,
         ...this.propertyGridUtils.guessValue(asset.dataSubcategory?.label),
       });
     }
     if (includeEmpty || asset.dataModel) {
       fields.push({
         icon: 'category',
         label: 'Data Model',
-        labelTitle: AssetProperties.dataModel,
         ...this.propertyGridUtils.guessValue(asset.dataModel),
       });
     }
     if (includeEmpty || asset.geoReferenceMethod) {
       fields.push({
         icon: 'commute',
         label: 'Geo Reference Method',
-        labelTitle: AssetProperties.geoReferenceMethod,
         ...this.propertyGridUtils.guessValue(asset.geoReferenceMethod),
       });
     }
     return fields;
   }
 
-  onShowPolicyDetailsClick(
-    title: string,
-    subtitle: string,
-    policyDetails: Policy,
-  ) {
-    const data: JsonDialogData = {
-      title,
-      subtitle,
-      icon: 'policy',
-      objectForJson: policyDetails,
-    };
-    this.matDialog.open(JsonDialogComponent, {data});
-  }
-
```

Comment:
was this not used by Broker?

Reply:
the function was inlined, and the jsonDialogService was used.